crates/payload/ethereum/src/lib.rs

```diff
@@ -313,7 +313,7 @@ where
             tx_type: tx.tx_type(),
             success: result.is_success(),
             cumulative_gas_used,
-            logs: result.logs().into_iter().map(Into::into).collect(),
+            logs: result.logs().iter().cloned().map(Into::into).collect(),
```

Comment:
nit do we need the clones? maybe doesn't matter perf wise

Reply:
replaced with `into_logs().into_iter()`